flamedisx/source.py

```diff
@@ -32,8 +32,12 @@ class Source:
     trace_difrate = True
 
     default_max_sigma = 3
+    default_max_sigma_outer = 3
     default_max_dim_size = 70
-    default_max_dim_size_outer = 120
+
+    # Capping the domain size for hidden variable dimensions. Any which aren't
+    # set will default to default_max_dim_size
+    max_dim_sizes: ty.Dict[str, int] = dict()
```

Comment:
Can you add a comment with how to give the `max_dim_sizes` for all the blocks both in the NEST sources and in the standard sources? 
An example of a dict would suffice.